src/index.html

```diff
@@ -77,13 +77,23 @@
                 name="routerName"
                 placeholder="{{'basicRouterSetup.routerName.placeholder' | translate }}"
                 data-ng-model="wizard.router.name"
+                data-ng-required="(wizard.internet.share && wizard.internet.vpn03.enabled && state.internet.vpn03.generate) || state.ip.register"
                 required pattern="[A-Za-z0-9_\-]{0,16}"
                 >
               <span class="fa form-control-feedback"
                 ng-class="formControlFeedback('routerName')"></span>
-              <p class="text-muted" translate>
+              <p class="text-muted" ng-hide="hasError('routerName')" translate>
```

Comment:
The help text also disappears when the `routerName` contains illegal characters.

Personally, I don't think that it's too much text if the error message and the description is shown. However, the description and error messages could be improved (and shortened!). Example:

* error.required: This field is required.
* help: The router name is used to identify your router in the Freifunk network with a human-readable name.

@geirkairam @k-nut: what do you think?

btw: when this PR is merged, we can consider using [ngMessages](https://docs.angularjs.org/api/ngMessages/directive/ngMessages) for displaying validation messages. It takes care of displaying only one validation message at a time.